mantidproject/mantid
# Add SaveReflections for various single crystal peak formats.

This adds an little python algorithm to save single crystal peaks in a variety of different formats. This brings together the new formats requested by scientists in the issue description and also integrates saving peaks to formats supported via `SaveHKL`

**To test:**
 - The unit tests are in place to check the formats output match what the various external program expect.
 - Check it works as expected. Try to break it with screwy input parameters.

Fixes #20920 

**Release Notes** 
See [here]()

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
